addons/l10n_sg/migrations/2.2/end-migrate_update_taxes.py

```diff
@@ -0,0 +1,6 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.account.models.chart_template import update_taxes_from_templates
+
+
+def migrate(cr, version):
+    update_taxes_from_templates(cr, 'l10n_sg.sg_chart_template')
```

Comment:
From saas~16.2 onwards try_loading method should be used.

Reply:
Hi thank you, we are aware of it it will be changed as soon as the dev as some time to look into it :smile: